guides/spin/comparison/index.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: single
+title:  "Spin and Roer + Managed Pipeline Templates"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+## Foreward
+
+This document is meant to describe the current state of managing
+pipeline and application config in Spinnaker 'as is', and describe
+a bit about future threads of work in this area.
+
+## `roer` and Managed Pipeline Templates
+
+### What it is
+
+Managed pipeline templates (MPT) is a capability in Spinnaker to
+formulate pipelines as an extended form of Jinja templates
+to allow for templating via variable bindings as well as
+inheritance. The spec for MPT is [here](https://github.com/spinnaker/dcd-spec/blob/master/PIPELINE_TEMPLATES.md).
+The template rendering is done server side in Orca.
+
+Pipeline templates themselves can be managed through Deck or through
+[`roer`](https://github.com/spinnaker/roer), a small CLI for manipulating
+pipeline templates.
+
+### What it isn't
+
+MPT and `roer` are scoped strictly to make managing Spinnaker pipelines
+easier. It is not a generic solution to make API calls to Spinnaker.
+
+### Where it's going
+
+MPT and `roer` are not going away anytime soon. Plenty of folks
+happily use `roer` and MPT.
+
+That being said, it's not actively being developed -- we are
+not adding new features.
+
+## `spin`
+
+### What it is
+
+`spin` is a CLI to interact with Spinnaker via the Gate API. It is
+currently in alpha, so the breadth of the interface is limited to managing
+pipeline and application config. Since `spin` is meant to be an API wrapper,
+it accepts JSON arguments and submits authenticated calls to Gate.
+
+It is meant to generically address the problem of automating flows that call the Spinnaker API.
+
+### What it isn't
+
+`spin` is not scoped strictly to solve the problem of managing Spinnaker
+pipelines. Submitting pipelines is only one of `spin`'s supported workflows.
+
+### Where it's going
+
+The surface area of `spin` is meant to grow with user demand of new automation flows.
+As an example, one prospective extension is to submit generic tasks to Gate
+for Orca to orchestrate.
+
+## Future Work
+
+We have gained valuable experience and insight from writing `roer`, MPT, and `spin`.
+
+We'll continue to grow `spin`'s adoption by expanding API coverage, hardening, and
+improving usability.
+
+We'll continue support for MPT and `roer`, while using the lessons we learned to
```

Comment:
Unless the plan changed, I would be clear here that this is bug-fix support -- the next generation of MPT & roer will likely be different tools